src/Collapse/Collapse.stories.tsx

```diff
@@ -8,7 +8,7 @@ storiesOf('Collapse', module).add(
   'Collapse',
   () => (
     <Collapse
-      isCollapsed={boolean('isCollapsed', true)}
+      isOpen={boolean('isCollapsed', true)}
```

Comment:
In storybook knobs, this will show the name of the prop as `isCollapsed` instead of `isOpen` which can be confusing. `isOpen={boolean('isOpen', true)}` is more direct

Reply:
Correct. I'll fix it in few minutes